content/components/index.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Components
+---
+
+import {Grid, Flex, Box, Link, Text, Label} from '@primer/components'
+
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 2fr']} gridGap={4}>
+    <div>
+        <Link href="/components/action-list">
+            <img alt="Select panel" src="https://user-images.githubusercontent.com/293280/123880841-54bb9280-d8f8-11eb-9a6f-4c33c8196fbf.png" />
+        </Link>
+    </div>
+    <div>
+        <Link href="/components/action-list" fontWeight="bold">Action list</Link>
+        <Box as="p">An action list is a vertical list of interactive actions or options. It’s composed of items presented in a consistent single-column format, with room for icons, descriptions, side information, and other rich visuals.</Box>
+    </div>
```

Comment:
From here down, we need to either comment out or move to another PR @vdepizzol

Reply:
Done!